CHANGELOG.md

```diff
@@ -1,3 +1,25 @@
+## [Release 1.37.4](https://github.com/aws-amplify/amplify-android/releases/tag/release_v1.37.4)
+
+### Miscellaneous
```

Comment:
Most of the items in this section (and the core-kotlin CHANGELOG) look like individual commits from PRs and those should be removed.

Reply:
Done